src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -502,6 +522,20 @@ public boolean onRunningTick(ItemStack aStack) {
         return true;
     }
 
+    private ItemStack getInputBusContent(int index) {
+        if (mItemCache[index] == NULL) {
+            ItemStack stuff = null;
```

Comment:
You should probably make stuff = NULL so that it doesn't go to null when the bus empties. Also i think using 'firstItemFound' as the variable name would be better.

Reply:
The purpose of a dedicated NULL is to signal this slot has been checked, not a protection against NPE. Having the stuff default to NULL defeats this purpose.